tests/integration_test.py

```diff
@@ -0,0 +1,49 @@
+import pytest
+
+import zarr
+import dask.array as dsa
+
+from rechunker import api
+
+
+@pytest.fixture
+def rechunk_delayed(tmp_path):
+    store_source = str(tmp_path / "source.zarr")
+    shape = (8000, 8000)
+    source_chunks = (200, 8000)
+    dtype = "f4"
+
+    a_source = zarr.ones(shape, chunks=source_chunks, dtype=dtype, store=store_source)
+
+    target_store = str(tmp_path / "target.zarr")
+    temp_store = str(tmp_path / "temp.zarr")
+    max_mem = 25600000
+    target_chunks = (8000, 200)
+    return (
+        api.rechunk_zarr2zarr_w_dask(
```

Comment:
The name of this function has changed to `_rechunk_array`. Or maybe you want to use the public function `rechunk`?

Reply:
Actually, maybe you want to move all your tests to the `test_rechunk.py` module?

Reply:
Yeah I messed up the git merge. `test_rechunk` should have everything now.